lib/ui/Editor.js

```diff
@@ -621,6 +626,14 @@ Editor.prototype._updateCursor = function () {
     self.screen.program.hideCursor();
   }
 };
+Editor.prototype._clipCursorToContent = function (cursor, content) {
```

Comment:
This function clips cursor to the current line size, not the whole file content ( which `getEndPosition` seems to return )